Patch the version and git commit into the scripts so that we don't need
to check out a git clone.
--- buildscript/build.go.orig	2021-07-12 18:46:40.000000000 -0500
+++ buildscript/build.go	2021-07-12 21:12:35.000000000 -0500
@@ -9,7 +9,6 @@
 	"path/filepath"
 	"strings"
 
-	"github.com/craiggwilson/goke/pkg/git"
 	"github.com/craiggwilson/goke/pkg/sh"
 	"github.com/craiggwilson/goke/task"
 	"github.com/mongodb/mongo-tools/common/testtype"
@@ -170,7 +169,7 @@
 		return "", fmt.Errorf("failed to get current version: %w", err)
 	}
 
-	gitCommit, err := git.SHA1(ctx)
+	gitCommit, err := "@GIT_COMMIT@", error(nil)
 	if err != nil {
 		return "", fmt.Errorf("failed to get git commit hash: %w", err)
 	}
--- release/version/version.go.orig	2020-11-11 14:35:13.000000000 -0600
+++ release/version/version.go	2020-11-28 08:23:14.000000000 -0600
@@ -62,12 +62,12 @@
 }
 
 func GetCurrent() (Version, error) {
-	commit, err := git("rev-parse", "HEAD")
+	commit, err := "@GIT_COMMIT@", error(nil)
 	if err != nil {
 		return Version{}, fmt.Errorf("git rev-parse HEAD failed: %w", err)
 	}
 
-	desc, err := git("describe", "--dirty")
+	desc, err := "@VERSION@", error(nil)
 	if err != nil {
 		return Version{}, fmt.Errorf("git describe --dirty failed: %w", err)
 	}
